src/SDKs/Search/DataPlane/Search.Tests/Utilities/EncryptionFixture.cs

```diff
@@ -0,0 +1,200 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+
+namespace Microsoft.Azure.Search.Tests.Utilities
+{
+    using Microsoft.Azure.KeyVault;
+    using Microsoft.Azure.KeyVault.Models;
+    using Microsoft.Azure.KeyVault.WebKey;
+    using Microsoft.Azure.Management.KeyVault;
+    using Microsoft.Azure.Management.KeyVault.Models;
+    using Microsoft.Azure.Management.ResourceManager;
+    using Microsoft.Azure.Management.ResourceManager.Models;
+    using Microsoft.Azure.Management.Search;
+    using Microsoft.Azure.Test.HttpRecorder;
+    using Microsoft.IdentityModel.Clients.ActiveDirectory;
+    using Microsoft.Rest.ClientRuntime.Azure.TestFramework;
+    using System;
+    using System.Collections.Generic;
+    using System.Net.Http;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using static Microsoft.Azure.KeyVault.KeyVaultClient;
+
+    public class EncryptionFixture : SearchServiceFixture
+    {
+        public string KeyVaultUri { get; private set; }
+
+        public string KeyVaultName { get; private set; }
+
+        public string KeyName { get; private set; }
+
+        public string KeyVersion { get; private set; }
+
+        public string TestAADApplicationId => "c74474ff-dae3-4e4a-be9a-6fa626a5e314";
+
+        public string TestAADApplicationObjectId => "55a05423-7beb-4a7b-8511-d355a08a28f8";
+
+        public string TestAADApplicationSecret => "G(#0i^]?[T&l?X!:p+(@>!*/.N)>*#)+_"; // [SuppressMessage("Microsoft.Security", "CS001:SecretInline", Justification = "This is a public secret with no assigned security role.")]
```

Comment:
As discussed offline, values related to the test account we use to run the test are in the set-sdktestaccount connection string, while values related to the registered AAD application used in the test are hard coded in the tests themselves (since those values are needed to run the test even in playback mode)